setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
+set -e
+
+HOMEBREW_NO_AUTO_UPDATE=1
+HOMEBREW_NO_INSTALL_CLEANUP=1
```

Comment:
I think we need to export those environment variables.

Reply:
We'd have to export them if they should be available in the parent shell and in this case they should only be available in this script (and the commands it's executing).

I think it's fine that way, but please let me know if I got it wrong.

Reply:
Ah, I wasn't sure if `cleanup.sh` and `package.sh` needed those environment variables too. You are right, these only apply to this shell (it's only needed for the `brew install` commands).

As an aside, the run duration of this workflow on GitHub Actions is quite fast compared to Travis CI (10 minutes versus 16 minutes). 👍